File: py/day4/day4_test.py
from day4 import missing_minimum_positive_integer

def test_day4_1():
    assert missing_minimum_positive_integer([3, 4, -1,1]) == 2

def test_day4_2():
    assert missing_minimum_positive_integer([1,2,0]) == 3

def test_day4_3():
    assert missing_minimum_positive_integer([-2,-3,-3,0]) == 1

    

File: js/day4/day4.test.js
const missingMinimumPositiveInteger = require("./day4");
describe("Day4", () => {
  test("given [3, 4, -1,1] return 2", () => {
    expect(missingMinimumPositiveInteger([3, 4, -1, 1])).toEqual(2);
  });
  
  test("given [1,2,0] return 3", () => {
    expect(missingMinimumPositiveInteger([1,2,0])).toEqual(3);
  });
  
  test("given [-4,-2,-2,0] return 1", () => {
    expect(missingMinimumPositiveInteger([-4,-2,-2,0])).toEqual(1);
  });
});
